src/main/java/org/gridsuite/explore/server/services/StudyService.java

```diff
@@ -138,4 +140,15 @@ public Flux<Map<String, Object>> getMetadata(List<UUID> studiesUuids) {
             .publishOn(Schedulers.boundedElastic())
             .log(ROOT_CATEGORY_REACTOR, Level.FINE);
     }
+
+    public Mono<String> getCaseImportParameters(UUID caseUuid) {
```

Comment:
Really necessary ?
No added value

Reply:
used by frontend to get import parameters, to display all fields that can be filled by users